public/js/cin.js

```diff
@@ -123,166 +132,159 @@ const PastScheduleCard = (props) => {
 };
 
 function prettifyCandidates(candidates) {
-  var names = [];
-  candidates.forEach((candidate) => {
-    names.push(candidate.full_name);
-  });
-  return names.join(", ");
+    var names = [];
+    candidates.forEach((candidate) => {
+        names.push(candidate.full_name);
+    });
+    return names.join(", ");
 }
 
 const getInterviewers = (event) => {
-  var names = [];
-  event.user_roles.forEach((user) => {
-    if (user.role.name == "Interviewer") {
-      names.push(user.user.name);
-    }
-  });
-  return names.join(", ");
+    var names = [];
+    event.user_roles.forEach((user) => {
+        if (user.role.name == "Interviewer") {
+            names.push(user.user.name + " (" + user.user.postnom + ")");
```

Comment:
People can not have postnoms
```suggestion
            names.push(user.user.name + (user.user.postnom.length > 0 ? (" (" + user.user.postnom + ")") : ""));
```